Improve README for Kafka example (#1885)
kanisterio/kanister: examples/kafka/adobe-s3-connector/README.md

$ kubectl create -f ./kafka-blueprint.yaml -n kanister
```

> ## Insert Data in Topic

Reviewer: should this be under `Setup Kafka` as H3?

Author: no

> * Create a topic `blogs` on Kafka server. The `blogs` topic is configured as source and sink topics in `s3config` configmap

Reviewer 2: ```suggestion
* Create a topic `blogs` on the Kafka server. The `blogs` topic is configured as a source and sink topic in `s3config` configmap
```

```bash
$ kubectl -n kafka-test run kafka-producer -ti --image=strimzi/kafka:0.20.0-kafka-2.6.0 --rm=true --restart=Never -- bin/kafka-topics.sh --create --topic blogpost --bootstrap-server my-cluster-kafka-bootstrap:9092
```

> * Create a producer to push data to `blogs` topic

Reviewer: is there any specific reason to change blogpost to blogs? If not, let's keep `blogposts`.

Author: Yeah, we need to change it, because in the configuration files we are using `blogs` as topics.

```bash
$ kubectl -n kafka-test run kafka-producer -ti --image=strimzi/kafka:0.20.0-kafka-2.6.0 --rm=true --restart=Never -- bin/kafka-console-producer.sh --broker-list my-cluster-kafka-bootstrap:9092 --topic blogs

>{"title":"The Matrix","year":1999,"cast":["Keanu Reeves","Laurence Fishburne","Carrie-Anne Moss","Hugo Weaving","Joe Pantoliano"],"genres":["Science Fiction"]}
>{"title":"ABCD3","year":2000,"cast":["Keanu Reeves","Laurence Fishburne","Carrie-Anne Moss","Hugo Weaving","Joe Pantoliano"],"genres":["Science Fiction"]}
>{"title":"Student of the year","year":2001,"cast":["Keanu Reeves","Laurence Fishburne","Carrie-Anne Moss","Hugo Weaving","Joe Pantoliano"],"genres":["Science Fiction"]}
>{"title":"ABCD","year":2002,"cast":["Keanu Reeves","Laurence Fishburne","Carrie-Anne Moss","Hugo Weaving","Joe Pantoliano"],"genres":["Science Fiction"]}
```

## Perform Backup

> To perform backup to S3, an ActionSet is created to run `kafka-connect`.

Reviewer 2: ```suggestion
To perform a backup to S3, an ActionSet is created which runs `kafka-connect`.
```

> To perform backup to S3, an ActionSet is created to run `kafka-connect`.
> ```bash

Reviewer: ```suggestion
To perform backup to S3, an ActionSet is created to run `kafka-connect`.

```bash
```

We should have a newline between text and code block.

# Create an actionset
$ kanctl create actionset --action backup --namespace kanister --blueprint kafka-blueprint --profile kafka-test/s3-profile-fn64h --objects v1/configmaps/kafka-test/s3config
```

### Disaster strikes!

> Let's say someone accidentally removed the events from `blogs` topic in the Kafka cluster:

Reviewer 2: ```suggestion
Let's say someone accidentally removed the events from the `blogs` topic in the Kafka cluster:
```

> ```bash

Reviewer 2: ```suggestion

```bash
```

* Before running pre-hook operation, confirm that no other consumer is consuming data from that topic

## Verify restore

> Create a consumer for topic

Reviewer: Let's add a newline after headings.
